project_partner_name/README.rst

```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Project Partner Name
+=========================
+
+This module extends the functionality of Project:
+
+* Makes "Customer" mandatory field.
+* Modifies the name_get adding the customer reference to the project name.
+* Modifies the name_search to search also by customer reference linked to project.
+
+Installation
+============
+
+To install this module, you need to:
+
+ * have basic modules installed (project)
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/Elico-Corp/elico_odoo/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/Elico-Corp/elico_odoo/issues/new?body=module:%20read_customer_reference%0Aversion:%20{8.0.1.0.1}%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Victor M. Martin <victor.martin@elico-corp.com>
+
+Maintainer
+----------
+
+.. image:: https://www.elico-corp.com/logo.png
+:alt: Elico Corp
+:target: https://www.elico-corp.com
+
+This module is maintained by Elico Corporation.
+
+Elico Corporation offers consulting services to implement open source management software in SMEs, with a strong involvement in quality of service.
```

Comment:
modify this footer with <80 char lines (and please inform everybody about the new footer)